src/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixer.php

```diff
@@ -91,7 +91,13 @@ public function configure(array $configuration = null)
                     $this->tokenKindCallbackMap[CT_USE_TRAIT] = 'removeBetweenUse';
                     break;
                 case 'curly_brace_open':
-                    $this->tokenEqualsMap['{'] = 'fixAfterToken';
+                    $this->tokenEqualsMap['{'] = 'fixAfterTokenIfMultiLineBody'; // i.e. not: CT_ARRAY_INDEX_CURLY_BRACE_OPEN
+                    break;
+                case 'square_brace_open':
+                    $this->tokenKindCallbackMap[CT_ARRAY_SQUARE_BRACE_OPEN] = 'fixAfterTokenIfMultiLineBody'; // typeless '[' tokens should not be fixed (to rare)
```

Comment:
Indeed typo in first case, good catch :+1: 
Your cases are correct related to the current state of the PR.
The last to sample could be fixed, but will take more time to develop and amend to this PR.
Let me think about those two to see how hard it would be (i.e. at config rule `after comma`, `before ] if within block`).